src/views/emails/template-parts/body/order/ticket-totals/coupons-row.php

```diff
@@ -0,0 +1,56 @@
+<?php
+/**
+ * Event Tickets Emails: Order Ticket Totals - Coupons Row
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/emails/template-parts/body/order/ticket-totals/coupons-row.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link https://evnt.is/tickets-emails-tpl Help article for Tickets Emails template files.
+ *
+ * @version TBD
+ *
+ * @since TBD
+ *
+ * @var Tribe__Template                    $this               Current template object.
+ * @var Email_Abstract $email              The email object.
+ * @var string                             $heading            The email heading.
+ * @var string                             $title              The email title.
+ * @var bool                               $preview            Whether the email is in preview mode or not.
+ * @var string                             $additional_content The email additional content.
+ * @var bool                               $is_tec_active      Whether `The Events Calendar` is active or not.
+ * @var \WP_Post                           $order              The order object.
```

Comment:
The spacing for these needs to be corrected. Same with the other new template files.

```suggestion
 * @var Tribe__Template $this               Current template object.
 * @var Email_Abstract  $email              The email object.
 * @var string          $heading            The email heading.
 * @var string          $title              The email title.
 * @var bool            $preview            Whether the email is in preview mode or not.
 * @var string          $additional_content The email additional content.
 * @var bool            $is_tec_active      Whether `The Events Calendar` is active or not.
 * @var \WP_Post        $order              The order object.
```